src/gradleinttest.sh

```diff
@@ -27,6 +27,13 @@ gradleinttest () {
   else
     echo "Skipping importing environment vars for gradleinttest"
   fi
+
+  # hardcode data project for connected tests
+  if [[ "${test_to_run}" == "testConnected" ]]; then
+    google_data_project="broad-jade-integration-data"
+    echo "Running ${test_to_run} test with data project: ${google_data_project}"
+  fi
```

Comment:
```suggestion
  if [[ "${test_to_run}" == "testConnected" ]]; then
    google_data_project="broad-jade-integration-data"
  else
    namespace_number=$(echo ${NAMESPACEINUSE} | sed 's/integration-//g')
    google_data_project="broad-jade-int-${namespace_number}-data"
  fi
  echo "Running ${test_to_run} test with data project: ${google_data_project}"
```

Reply:
would it be worth making this an input var for the action itself?

Reply:
I'm not sure -- what are the pros/cons of that? I'd defer to you on what's better, but would appreciate hearing your thinking.